### DISCUSSION TITLE
Detect transferring of repositories between two accounts from outside via API

### DISCUSSION THREAD

**Discussion by ALIENQuake** [community/community#24903] [Apps, API and Webhooks]
<p>Hi,</p>
<p>Actors:</p>
<p>Repo01 - name of the example repository</p>
<p>UserA - an example user with one “Repo01” repo, I’m not the owner</p>
<p>UnknowUserB - an example account name with, I’m not the owner and I dont know the name in advance</p>
<p>Scenario:</p>
<p>UserA use “Transfer Repository” feature to transfer Repo01 from himself to UserB</p>
<p>UnknowUserB accept “Transfer Repository” request and now “Repo01” is at <a href="http://github.com/UnknowUserB/Repo01" class="inline-onebox-loading" rel="nofollow noopener">github.com/UnknowUserB/Repo01</a></p>
<p>Is there any way to detect such operation via public API? So I could update extrenal repository list and put the new url there?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you know the original owner name and original repository name, you can find out where it went by using the <a href="https://developer.github.com/v3/repos/#get" rel="nofollow noopener">Get repository REST endpoint</a>. For example, I know that atom/tree-sitter-syntax was transferred to atom-archive/tree-sitter-syntax. If I access <a href="https://api.github.com/repos/atom/tree-sitter-syntax" rel="nofollow noopener">https://api.github.com/repos/atom/tree-sitter-syntax</a>, I’m redirected to <a href="https://api.github.com/repositories/67171757" rel="nofollow noopener">https://api.github.com/repositories/67171757</a>, which reveals that the repository full name is <code>atom-archive/tree-sitter-syntax</code> now.</p>
<p>I hope that helps!</p>
